tools/msstitch/msslookup_seqspace.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="msslookup_seqspace" name="Create lookup DB with sequences" version="1.1">
+    <description>from FASTA for filtering</description>
+    <requirements>
+        <requirement type="package" version="2.5">msstitch</requirement>
```

Comment:
You should probably use macros for all of these repetitive elements, citation, requirements a few params ...